sdk/tables/aztable/responses.go

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Samt thing here: there should not be a response.go file; the response structures should with the methods that need them.